spec/unit/stats/cleaner_spec.rb

```diff
@@ -0,0 +1,142 @@
+require_relative '../../spec_helper'
+
+module ThreeScale
+  module Backend
+    module Stats
+      describe Cleaner do
+        let(:storage) { Backend::Storage.instance }
```

Comment:
Would be nice to be able to mock Redis here, especially since these are unit tests (it's ok).

Reply:
Right. But we are not doing this in most of the other unit tests that we have, which in my opinion is something that we should solve, but not in this PR.

Reply:
It's ok 👍